DD/Views/Partials/homePage/_imageCarousel.cshtml

```diff
@@ -7,36 +7,46 @@
 }
 
 <head>
-    <link href="~/Styles/NewsCarousel.css" rel="stylesheet" />
+    <link href="~/Styles/ContentCarousel.css" rel="stylesheet" />
 </head>
 
 <div class="container">
     <div class="row justify-content-center">
-        <div class="col-8"> 
+        <div class="col-8">
             <div id="imageCarousel" class="carousel slide" data-bs-ride="">
+                <ol class="carousel-indicators">
+                    @for (int i = 0; i < blocksList.Count; i++)
```

Comment:
Try with for each instead if is not possible remember that arrays starts in 0 but count as validator can cause outbounds overflow exceptions.

Reply:
Just check